content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -68,8 +66,8 @@ Setting up a sidechain involves the following high-level steps:
     - [sidechain_federators] stanza - list of federators public keys to be used for signing. This list is common for all federators on the sidechain.
 4. Set up door accounts to enable cross chain transactions. This involves the following steps (on _both_ chains):
     - Create and fund the door accounts. 
-    - Set the signers list for the door accounts.
-    - Reserve 3 [tickets](tickets.html) for error handling.
+    - Set the ["SignerList"](set-up-multi-signing.html) for door accounts.
```

Comment:
```suggestion
    - [Set up the SignerList](set-up-multi-signing.html) for the door account.
```

- We don't use quotation marks for ledger object types such as SignerList.
- These bullet points are introduced as being needed on both chains. Since there's only one door account per chain, it should be singular here.
- The link more directly matches where it takes you to (["Set up Multi-signing" tutorial](https://xrpl.org/set-up-multi-signing.html)) if you include "Set up the" in the link text. If the link text is only "SignerList" then I would expect it to go to the [SignerList object reference](https://xrpl.org/signerlist.html).